Strategies/IITB_KPIs.md

```diff
@@ -0,0 +1,74 @@
+# IITB Key Performance Indicators (PKIs)
+
+IT Strategies are meant to move IITB (and ESDC) towards the end state [Vision of the Canadian Experience](../TeamMandate/Mandate.md). The following KPIs will be used to measure those IT Strategies successes. Monitoring those KPIs will be used to adjust strategies as needed to ensure efforts are well invested and remain aligned with [IITB's high level priorities](http://esdc.prv/en/iitb/corporate/Who_We_Are/Plans_and_Priorities/index.shtml) that are to:
+
+- Focus on the Core Business (of ESDC)
+- Modernize IM/IT Service Delivery
+- Be efficient in managing its operations
+- Be connected with IM/IT partners and suppliers
+
+## 1. **Percentage of ESDC Services consumable by APIs**
+
+ESDC services provided to the public or to other departments will need to be delivered digitally. This means via APIs. This transformation to digital will be measured by the rate at which analogue services are changed to digital ones.
+
+The indicator will be measured by using [ESDC's Service Catalogue](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) as reference point and monitor the rate at which they are transformed to digital
+
+`percentage = digital services / total services`
+
+## 2. **Rate of decrease in processing time of ESDC Services**
```

Comment:
1. The increase in personnel costs scenario is dealt with in 3. Reduction in toil

2. The increase in costs (large system deployment, expensive licences) is not covered, true. I see this as a financial matter that is part of the equation when designing services. The increase use of technology would probably also mean an increase ops costs for IITB due to the additional technology footprint across ESDC. Instead of having an indicator about op costs, I'd say 6. Increase rate of project intake by IITB be the indicator. This should counter large projects because if IITB can increase its rate of project intake, they'd have to be completed faster. Large projects would not.

Reply:
Sounds good!